CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# Contributors Guide
+
+## Bug Reports
+
+Please [open an issue](https://github.com/input-output-hk/cardano-sl/issues/new)
+to report about found bugs in Cardano SL.
+
+The more detailed your report, the faster it can be resolved and will ensure it
+is resolved in the right way.
+
+## Code
+
+If you would like to contribute code to fix a bug, add a new feature, or
+otherwise improve Cardano SL, pull requests are most welcome. It is a good idea to
+[submit an issue](https://github.com/input-output-hk/cardano-sl/issues/new) to
+discuss the change before plowing into writing code.
+
+Please make sure your contributions adhere to our coding guidelines:
+
+*  Code must adhere to the [Serokell Haskell Style Guide](https://github.com/serokell/serokell-util/blob/master/serokell-style.md).
+*  Code must be documented with [Haddock](https://www.haskell.org/haddock/doc/html/index.html).
+*  We are using GitFlow branching model, so pull requests need to be based on and opened against
+   the `develop` branch.
+
+Please note that Cardano SL uses a custom prelude [Universum](https://github.com/serokell/universum)
+instead of the default one.
+
+### Code Quality
+
+Cardano SL uses [HLint](https://github.com/ndmitchell/hlint) as a code quality tool.
+
+You can install it using `stack install hlint` command.
+
+To check Cardano SL code run this script (from the `cardano-sl` root directory):
+
+```
+$ ./scripts/haskell/lint.sh
+```
+
+### Code Style
+
+Cardano SL uses [stylish-haskell](https://github.com/jaspervdj/stylish-haskell) tool to prettify Haskell code.
+
+You can install it using `stack install stylish-haskell` command.
+
+To prettify Cardano SL code run this script (from the `cardano-sl` root directory):
```

Comment:
I don't think it should be in the guide for contributors. This script will apply `stylish-haskell` to the whole project. It's intended to be used rarely (e. g. when we modify stylish-haskell config file). So there is no point for contributors to use this script (we'll use it by ourselves when we find it necessary).
We can suggest configuring text editor to apply stylish-haskell automatically on save.